docs/reference/sql/endpoints/client-apps/tableau-desktop.asciidoc

```diff
@@ -0,0 +1,54 @@
+[role="xpack"]
+[testenv="platinum"]
+[[sql-client-apps-tableau-desktop]]
+=== Tableau Desktop
+
+Use the {es} JDBC driver and dedicated {es} Tableau Connector to access {es} data from Tableau Desktop.
+
+IMPORTANT: Elastic does not endorse, promote or provide support for this application; for native Elasticsearch integration in this
+product, please reach out to its vendor.
+
+[[sql-client-apps-tableau-desktop-prereqs]]
+==== Prerequisites
+
+* https://www.tableau.com/products/desktop[Tableau Desktop] 2019.4 or later
+* {es-sql} <<sql-jdbc, JDBC driver>>
+* https://www.elastic.co/downloads/tableau-connector[{es} Connector for Tableau]
+
+[[sql-client-apps-tableau-desktop-load-data]]
+==== Load data
+
+First, move or download the JDBC driver to the Tableau Desktop drivers directory:
+
+* Windows: `C:\Program Files\Tableau\Drivers`
+* Mac: `/Users/[user]/Library/Tableau/Drivers`
+
+Move the {es} Connector for Tableau to the Tableau Desktop connectors directory:
+
+* Windows: `C:\Users\[Windows User]\Documents\My Tableau Repository\Connectors`
+* Mac: `/Users/[user]/Documents/My Tableau Repository/Connectors`
+
+Launch Tableau Desktop. In the menu, click **More...** and select **Elasticsearch JDBC by
```

Comment:
I was under the impression that the new name is `Elasticsearch by Elastic`. Also, the print screen shows Elasticsearch by Elastic. So, no matter, what we choose as the name, it would be better if we are consistent throughout the wording and print-screens.